examples/text-sentiment/client.py

```diff
@@ -13,30 +13,54 @@
 # limitations under the License.
 
 # Third Party
+from start_runtime import protocol_arg
```

Comment:
Why isn't this using a relative import? Strange that it works

Reply:
there's no actual package to import! (no __init__.py in examples/text-sentiment)